blog/2024/09/juliacon-2025-announcement.md

```diff
@@ -0,0 +1,98 @@
+# JuliaCon Global 2025 and JuliaCon 2024 Wrap-Up
+
+![Pittsburgh Skyline](/assets/blog/2024-JuliaCon/AdobeStock_50245829.jpeg)
+We’re happy to announce that JuliaCon Global 2025 will be held in Pittsburgh Pennsylvania, USA in the ==third week of July==! As we approach the conference next year we will make further announcements.
+
+A few important date estimates for you:
+
+- Call for Proposals will be open November 1 - January 31
```

Comment:
Should add years to this, I think

Reply:
Can we provide an estimate for when acceptances will be out?

Reply:
Will we have a separate, earlier date for minis again this year?

Reply:
I would say Jan 15 first, with the optional extension later. I think acceptances should be out by Feb end.

Reply:
Can we get acceptances processed within January 22 - March 1?

Reply:
CfP this year closed on Feb 1st and notification of acceptance was sent on Feb 22nd, so doable I guess

Reply:
Okay sounds good I'll keep the dates currently listed unless y'all want to have a Program meeting briefly and discuss

Reply:
I left a more general comment in slack, but I think it's a bit orthogonal to the date themselves and does not prevent this from being published